StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/Helpers/OnSetterFunctionCalledMessageHandler.swift

```diff
@@ -8,35 +8,52 @@
 import Foundation
 import WebKit
 
-class OnSetterFunctionCalledMessageHandler<Values: Codable & Equatable>: ScriptMessageHandler<OnSetterFunctionCalledMessageHandler.Payload<Values>> {
+class OnSetterFunctionCalledMessageHandler: ScriptMessageHandler<VoidPayload> {
+    
+    class SetterHandler<Value: Codable & Equatable> {
+        let setter: String
+        let didReceiveMessage: (Value?)->Void
+        init(setter: String, didReceiveMessage: @escaping (Value?)->Void) {
+            self.setter = setter
+            self.didReceiveMessage = didReceiveMessage
+        }
+    }
```

Comment:
I like this solution. I am going to adapt it so we can add message handlers after init, since we want onExit to be handled at a different layer than onLoaderStart.